src/esp/sim/Simulator.cpp

```diff
@@ -577,6 +577,7 @@ void Simulator::reconfigureReplayManager() {
   gfxReplayMgr_->setPlayerCallback(
       std::bind(&esp::sim::Simulator::loadAndCreateRenderAssetInstance, this,
                 std::placeholders::_1, std::placeholders::_2));
+  // TODO: Is there a betterway than std::bind ?
```

Comment:
Like this?
```cpp
  gfxReplayMgr_->setPlayerCallback(
      [this](const assets::AssetInfo& assetInfo,
             const assets::RenderAssetInstanceCreationInfo& creation)
          -> scene::SceneNode* {
        return loadAndCreateRenderAssetInstance(assetInfo, creation);
      });
 ```

Reply:
@erikwijmans Yeah, they typing here won't allow an anonymous function though: https://github.com/facebookresearch/habitat-sim/blob/a3820c4ea7dad72026d8b2a6562349fd31b3ac8b/src/esp/gfx/replay/ReplayManager.h#L40

Reply:
That compiles fine for me.  A lambda should be fine for `std::function`.   Did you try the clang-tidy autogen fix and maybe that didn't work because it is using `auto&&` which then won't match?

Reply:
@erikwijmans Yeah, it compiled fine locally on my mac, but failed on GCC5 on CI.